### PR TITLE
Add docker support for cgroups V2

### PR DESCRIPTION
**Pull request by marcsanmi**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

Closes https://github.com/newrelic/nri-docker/issues/85.

## Give us some context

* What problems does this PR solve?

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Add support for cgroups V2 and refine a couple of metric descriptions.

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.

* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by sigilioso — DISMISSED** [review]

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx [hunk 1]
```diff
@@ -31,6 +31,10 @@ Requirement details for automatic Docker container monitoring for New Relic's in
 * If using CentOS, you must have CentOS version 6.0 or higher
 * Docker [version 1.12](https://www.docker.com/blog/docker-1-12-built-in-orchestration/) or higher
 
+<Callout variant="important">
+Support for operative systems using Control Group v2 is included from Infrastructure agent v.1.26.0 and nri-docker v1.7.0.
```
Comment (alvarocabanas) [inline src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx:35]: Operating systems

**Review by alvarocabanas — APPROVED** [review]

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx (hunk 1 above)
Comment (marcsanmi) [reply src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-instrumentation-infrastructure-monitoring.mdx]: Updated!

**Review by sigilioso — APPROVED** [review]

**Review by alvarocabanas — APPROVED** [review]
LGTM

**Review by x8a — APPROVED** [review]
Thanks!

**Pull request merged**